### PR TITLE
Migrate kafkasource to sources.knative.dev

### PR DESCRIPTION
**Pull request by lberk**
----

#

### CODE REVIEW

File: kafka/source/pkg/reconciler/controller.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 	sourcescheme "knative.dev/eventing-contrib/kafka/source/pkg/client/clientset/versioned/scheme"
 	kafkaclient "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/client"
 	kafkainformer "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/informers/sources/v1alpha1/kafkasource"
-	"knative.dev/eventing-contrib/pkg/apis/sources/v1alpha1"
```
Comment (matzew) [inline kafka/source/pkg/reconciler/controller.go]: why is that contrib in here ? :thinking:

Comment (lberk) [reply kafka/source/pkg/reconciler/controller.go]: it's not anymore? :)

**Comment by matzew** [comment]
/lgtm

```yaml
apiVersion: sources.knative.dev/v1alpha1
kind: KafkaSource
metadata:
  name: kafka-source
spec:
  consumerGroup: everythingisawesome
  bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
  topics: my-topic
  sink:
    apiVersion: serving.knative.dev/v1
    kind: Service
    name: event-display

```

tested w/ the above YAML

/assign @lionelvillard  for final approval

**Pull request merged**